RInAppMessaging/Classes/Views/Presenters/FullViewPresenter.swift

```diff
@@ -57,9 +57,8 @@ internal class FullViewPresenter: BaseViewPresenter, FullViewPresenterType {
         sendImpressions()
 
         if let unwrappedUri = sender.uri {
-            guard let uriToOpen = URL(string: unwrappedUri),
-                UIApplication.shared.canOpenURL(uriToOpen) else {
 
+            guard let uriToOpen = URL(string: unwrappedUri) else {
```

Comment:
Why does CodeCov think this isn't covered by tests?

Reply:
Hmm, it seems Codecov is right. No test reaches this line

Reply:
I added few tests for invalid and unsupported url but I didn't add any for positive scenarios because they are hard to test (mocking openURL doesn't make sense and normal openURL breaks other tests)